mailet/standard/src/test/java/org/apache/james/transport/mailets/RecipientToLowerCaseTest.java

```diff
@@ -0,0 +1,73 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.transport.mailets;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Collection;
+
+import org.apache.mailet.MailAddress;
+import org.apache.mailet.base.test.FakeMail;
+import org.assertj.core.api.iterable.Extractor;
+import org.junit.Before;
+import org.junit.Test;
+
+public class RecipientToLowerCaseTest {
+
+    public static final Extractor<MailAddress, String> AS_STRING = new Extractor<MailAddress, String>() {
+        @Override
+        public String extract(MailAddress mailAddress) {
+            return mailAddress.asString();
+        }
+    };
+
+    private RecipientToLowerCase testee;
+
+    @Before
+    public void setUp() {
+        testee = new RecipientToLowerCase();
+    }
+
+    @Test
+    public void serviceShouldPutRecipientToLowerCase() throws Exception {
+        FakeMail fakeMail = FakeMail.builder()
+            .recipient(new MailAddress("THienan1234@gmail.com"))
```

Comment:
No it's not (I asked them.)

Putting something close to their email is a way of "signing" their work